vllm/model_executor/layers/quantization/compressed_tensors/schemes/compressed_tensors_w8a8_int8.py

```diff
@@ -83,11 +98,18 @@ def create_weights(self, layer: torch.nn.Module,
                 output_partition_sizes, **layer_kwargs)
             layer.register_parameter("input_scale", input_scale)
 
+            if not self.input_symmetric:
```

Comment:
Why can't this be supported?

Reply:
It could be, it's just untested because I was told we don't care about that case

Reply:
Isn't this what this PR works on supporting?

Reply:
This PR adds support for dynamic per-token asymmetric quantized activations. I think this is more likely to be used than the static, per-tensor, asymmetric case, but do we have all the pieces implemented for that?

Reply:
We're only missing the integration piece for static asymmetric. ops & kernels are all tested supported and working

Reply:
Replaced with warning